internal/controller/volumegroup/volumegroupreplicationcontent.go

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright 2024 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package volumegroup
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	ctrl "github.com/ceph/ceph-csi/internal/controller"
```

Comment:
As this is a new file change the import order as per https://github.com/ceph/ceph-csi/blob/devel/docs/coding.md#imports